f/fcs/fn_fcsCommanderOverride.sqf

```diff
@@ -10,8 +10,17 @@ params ["_vehicle","_caller"];
 if (isNull cursorObject) then {
 	["NO TARGET",1] remoteExec ["f_fnc_fcsLocalWarning",commander _vehicle];
 } else {
-	// Get the position of the centre of the targeted object at the time of the override
-	_overrideTarget = (cursorObject modelToWorldWorld (boundingCenter cursorObject));
+	// Get the position of the targeted object at the time of the override
+	private _positionATL = getPosATL cursorObject;
```

Comment:
Perusing the wiki for `getPosATL` I noticed `getPosATLVisual`. Would using this instead maybe help with consistency, particularly with moving objects, since it'd be[^1] where the object "appears to be" instead of where it actually is

[^1]: if I understand the [wiki page on Render Time scope](https://community.bistudio.com/wiki/Simulation_vs_Render_Time_Scope) correctly

Reply:
The difference between render and simulation only really matters for high-speed objects and high-frequency checks. If we were tracking a moving object we might use render scope to ensure smooth tracking, but here we're just grabbing the position once. If it's moving slowly the difference won't be noticeable; if it's moving fast it'll be gone by the time the turret gets there anyway.